### PR TITLE
sync connect

### LINKED ISSUE #851: Sync connect
- [x] Run `bin/configlet sync -u -e connect --yes --docs --filepaths --metadata --tests include` (updates the Markdown files and maybe `tests.toml`)
  - docs: instructions unsynced: connect
  - metadata: unsynced: connect
- [x] Drop strict types comments from test file and example code (these are useless)
- [x] Add and sync test meta data to tests (`uuid` / `@testdox` in DocBlocks)
- [x] Decide on adding / adjusting / ordering test cases to match current problem specs

Do not redesign the student's interface or add test cases that would invalidate existing community solutions. These are extra tasks, which should be discussed in advance.

### PR DESCRIPTION
**Pull request by tomasnorre**
Resolves #851

### CODE REVIEW

**Comment by mk-mxp** [comment]
See #851 for things to decide before going on.

File: exercises/practice/connect/.meta/example.php [hunk 1]
```diff
@@ -118,40 +96,14 @@ public function blackStartCoords(): array
         }
         return $coords;
     }
-
-    // Prints the board, occasionally useful for debugging.
-    // Capital letters indicate a connect flag has been set.
-    public function dump(): void
```
Comment (tomasnorre) [inline exercises/practice/connect/.meta/example.php:124]: I allowed myself to remove this, as it's not needed for the solution.

**Comment by tomasnorre** [comment]
I hope I understood the feedback correctly.

**Review by mk-mxp — APPROVED** [review]
Well done! Thanks a lot for making this happen.

Merry christmas and a happy new year!

**Comment by tomasnorre** [comment]
> Well done! Thanks a lot for making this happen.
> 
> Merry christmas and a happy new year!

Thanks you too.

**Pull request merged**